### PR TITLE
Add polkit

### PR DESCRIPTION
**Pull request by satmandu**
- more gnome deps...
- hash build because meson was implemented

Works properly:
- [x] x86_64
- [x] i686
- [x] armv7l

### CODE REVIEW

**Review by supechicken — APPROVED** [review]

File: packages/polkit.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/polkit-0.118-265f-chromeos-x86_64.tar.xz'
+  })
+  binary_sha256({
+    aarch64: '6144024125f13a854cf48dce3d567a562beeeb6bfce82d01ae2309110a6ceaba',
+     armv7l: '6144024125f13a854cf48dce3d567a562beeeb6bfce82d01ae2309110a6ceaba',
+       i686: '6a3f46969a0ca97cd96ba1882ccc2adefe90e5dd31cd7020b0c4001c1af9137f',
+     x86_64: '3dd185d450d343bb6c4c1a073497fe50cd2afd1b380b8f682ee50ab7030177f5'
+  })
+
+  depends_on 'expat'
```
Comment (uberhacker) [inline packages/polkit.rb:24]: Core package?

File: packages/polkit.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  depends_on 'js78'
+  depends_on 'intltool' => ':build'
```
Comment (uberhacker) [inline packages/polkit.rb:26]: Core package?

**Comment by satmandu** [comment]
I really need to add a  dependency checker to my pre-compile preflight script.

**Comment by satmandu** [comment]
Truly amazing how simple that was.
```
for i in $(curl -Lsf https://github.com/skycocker/chromebrew/raw/master/tools/core_packages.txt); do sed -i "/depends_on '$i'/d" $1 ; done
```

**Pull request merged**